assets/js/services/adverts.service.js

```diff
@@ -21,16 +21,27 @@ function Adverts ($http, $rootScope) {
   }
 
   function fetch () {
-    let advertsFeed = $rootScope.rootURL + 'adverts_feed?wallet_version=3';
+    if (!$rootScope.apiDomain.endsWith('blockchain.info/')) {
+      return;
+    }
+
+    let advertsFeed = $rootScope.apiDomain + 'ads/get?wallet=true&n=2';
     $http.get(advertsFeed)
-      .success(data => {
-        let adverts = data.partners.home_buttons.splice(0);
-        service.ads.push(randFromArray(adverts));
-        service.ads.push(randFromArray(adverts));
-      });
-  }
+      .success(function (data) {
+        data.forEach(function (ad) {
+          var a = document.createElement('a');
+          a.href = ad.url;
```

Comment:
Let's not hard-code a URL into the ad feed; it's too tedious and risky to parse it. Instead, just construct the URL in the ad rendering code

So the JSON have `"id": "1234"`

And then the explorer / wallet would send the user to `$rootScope.apiDomain + "/ads/" + ad.id`